demo/demo_config.ts

```diff
@@ -1,5 +1,5 @@
-// The address of your Looker instance. Required.
-export const lookerHost = 'self-signed.looker.com:9999'
+// The address of your Looker instance. Required.  (Port must be included if it is required to reach the Looker server from browser clients, e.g. self-signed.looker.com:9999) 
+export const lookerHost = 'self-signed.looker.com'
```

Comment:
I don't think that if the example we're using is `self-signed.looker.com` that we should remove the port here. Maybe update the comment with other examples for a host server, but the way this is set up is to work with a freshly bootstrapped test instance.